src/step_definitions/screenshots.ts

```diff
@@ -0,0 +1,52 @@
+import { When, Then } from 'cucumber';
+import config from '../core/config.helper';
+import { promises as fsPromises } from 'fs';
+import { resolve } from 'path';
+
+const diffExists = async screenshotName => {
+  return await fsPromises
+    .access(
+      resolve(
+        `${config.imageComparator.temporaryFolder}/diff`,
+        `${screenshotName}-${config.browserWidth}x${config.browserHeight}.png`
+      )
+    )
+    .then(() =>
+      Promise.reject(
+        `Check "diff" catalog. Devation tolerance was bigger then: ${config.imageComparator.saveAboveTolerance}%`
+      )
+    )
+    .catch(() => Promise.resolve());
+};
+
+When(/^I take screenshot of the element "([^"]*)" and save as a "([^"]*)"$/, async function(
+  elementName,
+  screenshotName
+) {
+  await browser.imageComparison.saveElement(this.currentPage.getElement(elementName), screenshotName);
+});
+
+When(/^I take screenshot of the visible part of the page and save as a "([^"]*)"$/, async screenshotName => {
+  await browser.imageComparison.saveScreen(screenshotName);
```

Comment:
As far as i know you can return promise from cucumber step, so it's better to use return <promise> instead of await <promise> because of performance